operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/InformerStoppedHandler.java

```diff
@@ -0,0 +1,9 @@
+package io.javaoperatorsdk.operator.api.config;
+
+import io.fabric8.kubernetes.client.informers.SharedIndexInformer;
+
+public interface InformerStoppedHandler {
```

Comment:
Would rather rename it to `InformerStopHandler`

Reply:
I think the current name is better because it reflects the fact that the handler is called *after* the informer is stopped.